test/truefi2/TrueCreditAgency.test.ts

```diff
@@ -261,17 +261,86 @@ describe('TrueCreditAgency', () => {
     )
   })
 
-  describe('Repaying', () => {
+  describe('payInterest', () => {
     beforeEach(async () => {
       await creditAgency.allowBorrower(borrower.address, true)
+      await creditAgency.setRiskPremium(1000)
+      await creditOracle.setScore(owner.address, 255)
+      await creditAgency.connect(borrower).borrow(tusdPool.address, 1000)
+      await tusd.connect(borrower).approve(creditAgency.address, 1000)
+      await timeTravel(YEAR)
+    })
+
+    it('pays interest to the pool', async () => {
+      await creditAgency.connect(borrower).payInterest(tusdPool.address)
+
+      expect(await tusd.balanceOf(borrower.address)).to.be.closeTo(BigNumber.from(900), 2)
+      expect(await tusd.balanceOf(tusdPool.address)).to.be.closeTo(parseEth(1e7).sub(900), 2)
+    })
+
+    it('increases paidTotalInterest', async () => {
+      await creditAgency.connect(borrower).payInterest(tusdPool.address)
+      expect(await creditAgency.paidTotalInterest(tusdPool.address, borrower.address)).to.be.closeTo(BigNumber.from(100), 2)
+    })
+
+    it('pays close to nothing on second call', async () => {
+      await creditAgency.connect(borrower).payInterest(tusdPool.address)
+      await creditAgency.connect(borrower).payInterest(tusdPool.address)
+      expect(await creditAgency.paidTotalInterest(tusdPool.address, borrower.address)).to.be.closeTo(BigNumber.from(100), 2)
```

Comment:
This test doesn't say much without the context of the previous one. You can either add same expect after first payment or extract the amount from the event